scanners/flutter/flutter.go

```diff
@@ -179,6 +179,14 @@ projects:
 			proj.hasAndroidProject = true
 		}
 
+		// todo: duplicate code; factor into detectAndroidProjectPath function?
```

Comment:
This is somewhat similar: https://github.com/bitrise-tools/go-android/blob/master/gradle/project.go

Reply:
Thanks, will discuss offline if it is usable here. TODO comment will be removed before merge.